examples/resources/btp_globalaccount_api_credential/import.sh

```diff
@@ -0,0 +1,3 @@
+# terraform import btp_globalaccount_api_credential.<resource_name> <globalaccount_id>
```

Comment:
The comment in the import should contain the full key for import. It seems that here this is only a part.
If it is a mixed key, then the different components should be comma.-separated without a space. See e.g. <https://registry.terraform.io/providers/SAP/btp/latest/docs/resources/directory_role_collection#import>